src/helpers/git_helpers/branch_configuration_helpers.sh

```diff
@@ -94,89 +123,72 @@ function echo_update_child_branches {
 
 # Makes sure that we know all the parent branches
 # Asks the user if necessary
-# Aborts the script if not all branches become known.
+#
+# branches - space seperated list of branches
 function ensure_knows_parent_branches {
-  local current_branch=$1
-
-  if [ "$(knows_all_ancestor_branches "$current_branch")" = true ]; then
-    return
-  fi
-
-  # Here we don't have the ancestors list --> make sure we know all ancestors, then recompile it from all ancestors
+  local branches=$1
+  local branch
+  local child
   local parent
+  local user_input
+  local numerical_regex='^[0-9]+$'
```

Comment:
Do you want to keep the list of local variable declarations sorted alphabetically, or grouped by assignment vs no assignment?